luxonis_train/models/luxonis_model.py

```diff
@@ -620,18 +622,23 @@ def configure_callbacks(self) -> list[pl.Callback]:
         self.best_val_metric_checkpoints_path = f"{self.save_dir}/best_val_metric"
         model_name = self.cfg.model.name
 
-        callbacks: list[pl.Callback] = []
+        user_callbacks = [c.name for c in self.cfg.trainer.callbacks]
 
-        callbacks.append(
+        callbacks: list[pl.Callback] = [
             ModelCheckpoint(
                 monitor="val/loss",
                 dirpath=self.min_val_loss_checkpoints_path,
                 filename=f"{model_name}_loss={{val/loss:.4f}}_{{epoch:02d}}",
                 auto_insert_metric_name=False,
                 save_top_k=self.cfg.trainer.save_top_k,
                 mode="min",
-            )
-        )
+            ),
+        ]
+        if "DeviceStatsMonitor" not in user_callbacks:
```

Comment:
Should we have those enabled by default?

Reply:
They are. But the user can still define them themselves in which case they override the default.